### PR TITLE
Extract sources service

### PR DESCRIPTION
**Pull request by agrare**
Depends on: https://github.com/ManageIQ/topological_inventory-core/pull/121

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ gem 'rails',              '>= 5.2.2.1', '~> 5.2.2'
 
 gem 'inventory_refresh', :git => 'https://github.com/ManageIQ/inventory_refresh', :branch => 'master'
 gem 'manageiq-api-common', :git => 'https://github.com/ManageIQ/manageiq-api-common', :branch => 'master'
-gem 'topological_inventory-core', :git => 'https://github.com/ManageIQ/topological_inventory-core', :branch => 'master'
+gem 'topological_inventory-core', :git => 'https://github.com/agrare/topological_inventory-core', :branch => 'extract_sources_service'
```
Comment (agrare) [inline Gemfile]: Just to show tests passing on travis, remove before merge

**Comment by agrare** [comment]
@gtanzillo @abellotti please review

Want to get this lined up to be able to be merged when we're ready

**Comment by abellotti** [comment]
@agrare re: openapi-3-v1.0.0.json I checked out the branch and ran openapi:generate, the Sources definition has the following changes:

```
diff --git a/public/doc/openapi-3-v1.0.0.json b/public/doc/openapi-3-v1.0.0.json
index 71e41b0..ada1de4 100644
--- a/public/doc/openapi-3-v1.0.0.json
+++ b/public/doc/openapi-3-v1.0.0.json
@@ -3923,6 +3923,12 @@
           "id": {
             "$ref": "#/components/schemas/ID"
           },
+          "name": {
+            "type": "string"
+          },
+          "source_type_id": {
+            "$ref": "#/components/schemas/ID"
+          },
           "tenant_id": {
             "$ref": "#/components/schemas/ID"
           },
@@ -3935,6 +3941,9 @@
             "format": "date-time",
             "readOnly": true,
             "type": "string"
+          },
+          "version": {
+            "type": "string"
           }
         }
       }
```

We should reflect latest (i.e. openapi:generate should not show changes with 1.0), wdyt ?

**Comment by agrare** [comment]
@abellotti did you run the db migrations?  I ran openapi:generate and don't see any changes.

**Comment by abellotti** [comment]
Ahhh, that was it 👍 

Thanks @agrare, this LGTM!!

**Pull request merged**